drivers/hmis/app/models/hmis/role.rb

```diff
@@ -99,13 +99,23 @@ def self.global_permissions
   def self.permissions_by_group
     {}.tap do |perms|
       permissions_with_descriptions.each do |key, role|
+        # proc is optional; if it evaluates to false, then hide this permission.
+        # (If it's nil, still include the permission)
+        next if role[:proc] == false
+
         perms[role[:category]] ||= {}
         perms[role[:category]][role[:sub_category]] ||= {}
         perms[role[:category]][role[:sub_category]][key] = role
       end
     end
   end
 
+  def self.show_ce_permissions?
+    # Show CE permissions if CE is enabled
+    # In test env on CI, permissions_by_group is called before the database is loaded(?)
```

Comment:
https://github.com/greenriver/hmis-warehouse/actions/runs/15138463065/job/42555895992

to discuss/pair

Reply:
Proc shouldn't evaluate when the file is loaded, it should be evaluated at runtime. Like a lambda